components/brave_wallet_ui/constants/types.ts

```diff
@@ -1057,3 +1057,21 @@ export type RewardsExternalWallet = Pick<
 > & {
   status: WalletStatus
 }
+
+type SwapProviderNameType = {
+  [provider: BraveWallet.SwapProvider]: string
+}
```

Comment:
Do we need this type? It resolves to ```{ [provider: number]: string }``` which is not very strict

Reply:
Removed!